clusterutil/cluster_validation_config.go

```diff
@@ -54,10 +83,10 @@ type ClusterValidationProtocolConfig struct {
 	SoftValidation bool `yaml:"soft_validation" category:"experimental"`
 }
 
-func (cfg *ClusterValidationProtocolConfig) Validate(prefix string, label string) error {
-	if label == "" {
+func (cfg *ClusterValidationProtocolConfig) Validate(prefix string, labels []string) error {
+	if len(labels) == 0 {
```

Comment:
I think this is wrong. We want to validate that the "label" is set, we don't really care about the additional ones